src/components/Root/Root.js

```diff
@@ -165,6 +165,7 @@ class Root extends Component {
                   messages={translations}
                   textComponent={Fragment}
                   onError={config?.suppressIntlErrors ? () => {} : undefined}
+                  onWarn={config?.suppressIntlErrors ? () => {} : undefined}
```

Comment:
Hm, if you made it `suppressIntlWarnings` here wouldn't that suffice to give us  the option of suppressing errors, warnings, or both?